### PR TITLE
Create easy-sangria.json

### PR DESCRIPTION
**Pull request by xtbe**
Add recipe for easy Sangria

### CODE REVIEW

File: src/recipes/easy-sangria.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      
+    ],
+    "directions": [
+        "Put the chopped fruit in a bowl and sprinkle over the sugar and cinnamon, then stir to coat. ",
+        "Cover and leave to macerate in the fridge for at least 1 hr, or ideally overnight.",
+        "Fill a large jug with ice.",
+        "Stir the macerated fruit mixture to ensure the sugar is dissolved, then tip into the jug with the wine and brandy.",
+        "Stir, then top up with the sparkling water and serve."
+    ],
+    "image": "sangria-new-crop-8fbb7a1.jpg",
```
Comment (alfg) [inline src/recipes/easy-sangria.json:70]: Hi, thanks for the PR. It seems the image is missing. Also, can you rename this image to match the json filename too? `easy-sangria.jpg`. Thank you!

**Review by alfg — CHANGES_REQUESTED** [review]

File: src/recipes/easy-sangria.json (hunk 1 above)
Comment (alfg) [reply src/recipes/easy-sangria.json]: Hi, the image still seems to be missing. Can you add it in?

**Comment by xtbe** [comment]
Hm must have done something wrong then, I do see my second commit... which should fix the image...

**Comment by alfg** [comment]
Closing. Feel free to re-open with fixed image. Thanks!

**Comment by xtbe** [comment]
I added the missing image in a 2nd commit. Did I do something wrong in contingent the fix for this?

**Comment by alfg** [comment]
Hi @xtbe, I see the last commit as:
https://github.com/alfg/opendrinks/pull/721/commits/ca1d472c70fe01e4f33f9533f0a77038b4dfccd8

However, the image itself doesn't seem to be checked in.

**Comment by xtbe** [comment]
Thanks, so it looks like the image needs to be attached, thought it could be externally referenced. I will submit a new PR. Thank you for clarifying

**Pull request closed without merging**